app/endpoints/campaign.py

```diff
@@ -492,15 +577,23 @@ async def reset_vote(
 async def vote(
     vote: schemas_campaign.VoteBase,
     db: AsyncSession = Depends(get_db),
-    user: models_core.CoreUser = Depends(is_user_a_member_of(GroupType.AE)),
+    user: models_core.CoreUser = Depends(is_user_a_member),
 ):
     """
     Add a vote for a given campaign list.
 
     An user can only vote for one list per section.
 
-    **The user must be a member of the group AE to use this endpoint**
+    **The user must be a voter to use this endpoint**
     """
+    voters = await cruds_campaign.get_voters(db)
+    voters_groups = [voter.group_id for voter in voters]
+    if not voters_groups or not is_user_member_of_an_allowed_group(user, voters_groups):
```

Comment:
We either want to allow everyone to vote in `get_lists` or nobody here, when `voters_groups` is empty
```suggestion
    if not is_user_member_of_an_allowed_group(user, voters_groups):
```